pdfarranger/pdfarranger.py

```diff
@@ -267,6 +268,38 @@ def __init__(self, *args, **kwargs):
             # "private" clipboard does not work in Windows so we can't use it here
             self.clipboard_pdfarranger = Gtk.Clipboard.get(Gdk.Atom.intern('_SELECTION_PDFARRANGER',
                                                                            False))
+        if hasattr(self, "add_main_option"):
+            self.add_arguments()
+        else:
+            print(_(
+                "GLib.add_main_option missing, "
+                "was added in GLib 2.40 (Released 2018-09-22)"),
+                file=sys.stderr)
+
+    def add_arguments(self):
+        if hasattr(self, "set_option_context_summary"):
+            self.set_option_context_summary(_(
+               "PDF Arranger is a small python-gtk application, which helps the "
+               "user to merge or split pdf documents and rotate, crop and rearrange "
+               "their pages using an interactive and intuitive graphical interface ."
+               "It is a frontend for pikepdf.\n"
+               "Currently the only CLI option is -v, which displays the version number."
```

Comment:
The changes that I made break that functionality. Could you give me a pointer on where in the code you implemented that so that can add it back and remove dead code? (I don't find it on my own..)

Reply:
https://github.com/pdfarranger/pdfarranger/blob/65b651e3e689acfefcc0b6e928990cb651d6277e/pdfarranger/pdfarranger.py#L271

Reply:
> Moreover, the recognized binary name in the `--help` is `__main__.py`. I will have to correct that before it is merged.

This could also depend on how exactly PDF arranger ist launched. If you find a good solution that's fine but I wouldn't spend too much time or code on it. Hardcoding would be fine, too.

Reply:
Fixed with f7463c32f159c777dc15211e6f6b8cb007f9d705. I also added error handling in case of non-existant files, which was kind of implemented before:  the first file was correct but no the second, no error message was displayed and the canevas appeared empty.
After looking at the manpages of evince, totem, etc of which none talk about optional files, I think it is okay as is. `[FILES]` is displayed nonetheless at the very beginning: `Usage:  __main__.py [OPTION…] [FILES]`